voc/related.go

```diff
@@ -41,9 +41,10 @@ func (v VirtualMachine) Related() []string {
 		list = append(list, string(b))
 	}
 
-	if v.BootLogging != nil {
-		list = append(list, v.BootLogging.Related()...)
-	}
+	// Boot/os logging in Azure points to a URL, not a Resource ID, this is broken!
```

Comment:
I guess this change caused some tests to fail (see build error)

Reply:
Ah good point. Should I remove it from the Azure discoverer then? Basically the discoverer is doing it incorrectly.